### ISSUE TITLE
Clear up react test warnings

### ISSUE THREAD

**Issue by wldcordeiro** [devtools-html/debugger.html#4221]
When you run the test suite we get lots of warnings because of some deprecations and stuff over in React. We should update the appropriate packages and import paths.

```
  ● Console

    console.warn node_modules/react-addons-test-utils/index.js:31
      Warning: ReactTestUtils has been moved to react-dom/test-utils. Update references to remove this warning.
    console.error node_modules/fbjs/lib/warning.js:33
      Warning: Shallow renderer has been moved to react-test-renderer/shallow. Update references to remove this warning.
```

**Comment by pradeepgangwar** [comment]
I would like to work on this. Please help me how should i proceed.

**Comment by jasonLaster** [comment]
@pradeepgangwar thanks. Try running the debugger locally and see if you can reproduce the errors

**Comment by jasonLaster** [comment]
![Screen Shot 2017-10-02 at 12.18.07 AM.png](https://shipusercontent.com/c2ebcae71e7fe2652bea804018da212a/Screen%20Shot%202017-10-02%20at%2012.18.07%20AM.png)


I think a good way to reproduce is to run `yarn test`

**Comment by pradeepgangwar** [comment]
I also got same set of errors on running test suite by `yarn test-all`. So how should i proceed further to correct this.
![debugger-err](https://user-images.githubusercontent.com/21259802/31064527-a9741118-a75a-11e7-973b-99c251c60a6d.png)

**Comment by jasonLaster** [comment]
@pradeepgangwar i'm not sure. i'd look up the errors in google and try to track down the code that causes them

**Comment by wldcordeiro** [comment]
Looking further into this I believe it may be an issue in Enzyme itself. I'm digging further in and reporting it in their repo. Marking not-available for now till I get a bit more info.

**Comment by ahuth** [comment]
Picked up and resolved many of the warnings in #4248.

There are a few more for `PropTypes` and `createClass` that will take some more work to resolve.

**Comment by wldcordeiro** [comment]
@ahuth the `PropType` ones will be silenced soon since we're going full with Flow. As for `createClass` I assume it maybe coming from `devtools-core` as searching for `createClass` in the debugger only has a handful of results in docs.
